rfcs/032-lineage-backend.md

```diff
@@ -0,0 +1,97 @@
+- Feature Name: lineage-backend
+- Start Date: 2021-03-31
+- RFC PR: [amundsen-io/rfcs#32](https://github.com/amundsen-io/rfcs/pull/32)
+- Amundsen Issue: [amundsen-io/amundsen#0000](https://github.com/amundsen-io/amundsen/issues/0000) (leave this empty for now)
+
+# Amundsen Lineage - Backend
+
+## Summary
+
+Currently, we link to the 3rd Party lineage tool in order to show table and column lineage for tables in Amundsen. 
+The goal of this project is to have a lineage graph that is native to Amundsen so that users can find table and column lineage in one platform.
+
+## Motivation
+
+At the moment we are relying completely on any 3rd party lineage tool to show the table and column lineage in Amundsen. 
+Not everyone has access to such a 3rd party tool and want to have lineage information available natively in Amundsen.
+
+ref: https://github.com/amundsen-io/rfcs/blob/master/rfcs/025-lineage-stage-0.md
+
+## Guide-level Explanation (aka Product Details)
```

Comment:
could you describe whether backend is only for graph or atlas or mysql? which one does it not plan to support?

Reply:
added